openapi.yml

```diff
@@ -55,16 +93,81 @@ components:
       properties:
         id:
           type: string
+        category:
```

Comment:
I would maybe distinguish here between content type and physical format, e.g.:
```
content_type:
    description: High level categorization of the type of content, such as text, still image, audio, etc.
    type: string
format:
    description: High level categorization of the content format, such as online resource, CD, book, etc.
    type: string
```
Outstanding question about how specific we want to get with content types...

Reply:
I'll make that change if that's what the ontologists are going with for sure. Thanks.

Reply:
It's standard practice in all metadata schemas, even super high level ones like Dublin Core. But we can wait to make the change until we've decided exactly which terms to use.

Reply:
Nah, I'll make your suggested changes for now and we can tweak if we need to later. 👍